packages/core-macro/src/props/mod.rs

```diff
@@ -1769,6 +1769,17 @@ fn looks_like_write_type(ty: &Type) -> bool {
     }
 }
 
+fn looks_like_store_type(ty: &Type) -> bool {
+    match extract_base_type_without_generics(ty) {
+        Some(path_without_generics) => {
+            path_without_generics == parse_quote!(dioxus_stores::prelude::Store)
+                || path_without_generics == parse_quote!(prelude::Store)
+                || path_without_generics == parse_quote!(Store)
+        }
+        None => false,
+    }
+}
+
 fn looks_like_callback_type(ty: &Type) -> bool {
     let type_without_option = remove_option_wrapper(ty.clone());
     match extract_base_type_without_generics(&type_without_option) {
```

Comment:
same comment from the base PR here - maybe it'd make sense to add a check if the final path segment is `Store`?